### PR TITLE
Update zenodo.json

### PR DESCRIPTION
**Pull request by TheDragon246**
Solves https://github.com/adswa/git-annex-remote-osf/issues/14
will further update as details are provided.

### CODE REVIEW

**Comment by adswa** [comment]
Who is still missing? @mjboos @SRSteinkamp @nbeliy @kousu ..? Can you provide @TheDragon246  with your details, please?

File: .zenodo.json [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
+      "name": "Poldrack, Benjamin",
+      "orcid": "0000-0001-7628-0801"
+    },
+    {
+      "affiliation": "Psychoinformatics Lab, INM-7, Research Centre Juelich",
+      "name": "Wagner, Adina S.",
+      "orcid": "0000-0003-2917-3450"
+    },
+    {
+      "affiliation": "Erasmus University Rotterdam",
```
Comment (DorienHuijser) [inline .zenodo.json:19]:
```suggestion
      "affiliation": "Erasmus School of Social and Behavioral Sciences, Erasmus University Rotterdam",
```

**Comment by adswa** [comment]
Repeating for more pressure: Who is still missing? @mjboos @SRSteinkamp @nbeliy @kousu ..? Can you provide @TheDragon246 with your details, please?

:wink:

**Comment by adswa** [comment]
Moritz Boos
https://orcid.org/0000-0002-1219-4901
Carl von Ossietzky Universität Oldenburg

**Comment by SRSteinkamp** [comment]
Sorry for  the late reply  - for me it would be:

Simon R. Steinkamp
Cognitive Neuroscience, Institute of Neuroscience & Medicine (INM‐3), Research Centre Juelich, Juelich, Germany
https://orcid.org/0000-0002-6437-0700

Thanks!

**Comment by TheDragon246** [comment]
updated with current additions.

**Comment by mih** [comment]
@kousu is still missing!

**Comment by adswa** [comment]
Zenodo-linkage is switched on - a release here will get us a doi and zenodo entry

**Comment by mih** [comment]
So let's bring those changes in?

**Comment by TheDragon246** [comment]
It's updated with all available info till now. @kousu still missing unfortunately.

**Comment by mih** [comment]
Thx!

**Pull request merged**